datasets/cellpainting-gallery.yml

```diff
@@ -38,13 +47,61 @@ DataAtWork:
       URL: https://github.com/carpenterlab/2016_bray_natprot/wiki
       AuthorName: Carpenter-Singh Lab
```

Comment:
For web resources, I'd favor dropping `AuthorName` in favor of `AuthorURL`, because the latter is more accurate (at least in the examples we have here).

Reply:
If we must fill `AuthorName `, I'd go with "Multiple"